src/schema.ts

```diff
@@ -1330,6 +1332,7 @@ export const typeDefs = gql`
     owner: ArenaUserV2
     topicId: Int!
     flags: [ArenaFlag!]
+    replies: [ArenaPostV2!]!
```

Comment:
Denne må vel også ha toPid

Reply:
Den trenger ikke det! Vi caster om i NDLA frontend for å håndtere bytte til ArenaPostV2. I MyNDLA-api så er toPid kun en domain type!